pom.xml

```diff
@@ -295,6 +295,7 @@
         <log4j.version>2.11.2</log4j.version>
         <slf4j.version>1.7.26</slf4j.version>
         <metrics.version>3.2.6</metrics.version>
+        <metrics.jvm.version>4.0.7</metrics.jvm.version>
```

Comment:
Will there be any possible conflicts since it's pulling in two different major versions of dropwizard.metrics?
Even though they are for different artifacts, I am worried about transitive dependencies

Reply:
I didn't see any problems when running a mvn dependency tree.

[INFO] +- io.dropwizard.metrics:metrics-jvm:jar:4.0.7:compile
[INFO] |  +- (io.dropwizard.metrics:metrics-core:jar:3.2.6:compile - version managed from 4.0.7; omitted for duplicate)

Reply:
So there are two metrics-core in the dependencies, one is 4.0.7 and the other one is 3.2.6. I am afraid they could conflict each other at runtime since they are different at major version level.

Reply:
I'll change the version.